examples/bind-shopping.amp.html

```diff
@@ -0,0 +1,454 @@
+<!doctype html>
```

Comment:
> Not sure what you mean by onclick, did you mean the on attribute? Why is that a problem?

The `onclick` is an example of an AMP attribute that we do not allow for security reasons. Since we don't allow it in our static validator, we should also not allow it from binding resolutions.